src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,116 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.UserValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private static final String WRONG_LOGIN_ZERO_SYMBOLS = "";
+    private static final String MINIMUM_CORRECT_LOGIN_ONE_SYMBOL = "a";
+    private static final String ANOTHER_CORRECT_LOGIN = "aa";
+    private static final String FIVE_SYMBOLS_WRONG_PASSWORD = "five5";
+    private static final String MINIMUM_CORRECT_PASSWORD_SIX_SYMBOLS = "six456";
+    private static final int MINIMUM_CORRECT_YEARS_OLD = 18;
+    private static final int WRONG_YEARS_OLD = 17;
+    private static final User user = new User();
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user.setLogin(MINIMUM_CORRECT_LOGIN_ONE_SYMBOL);
+        user.setPassword(MINIMUM_CORRECT_PASSWORD_SIX_SYMBOLS);
+        user.setAge(MINIMUM_CORRECT_YEARS_OLD);
+    }
+
+    @Test
+    void getUserByLogin_ok() {
```

Comment:
For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; For example, if we are testing the method register with a null
user's age the test method name should be r`egister_nullAge_notOk. notOk` is because the test expects the register method to throw an exception.